rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -224,7 +224,7 @@ class ParameterVariant
     return parameter;
   }
 
-  std::string to_string() const
+  std::string value_to_string() const
```

Comment:
From the meeting @dirk-thomas will make a more general ticket about this design issue for us to have a standard approach.